docs/api/structures/message-dialog-details.md

```diff
@@ -0,0 +1,6 @@
+# MessageDialogDetails Object
+
+* `type` string - Can be `alert` or `confirm`.
+* `message` string - Dialog message text.
+* `frameProcessId` Integer
+* `frameRoutingId` Integer
```

Comment:
We should expose a `WebFrameMain` reference here instead of the IDs; see the `context-menu` event which passes a `frame` parameter.

Reply:
This comment seems unaddressed.

Reply:
Done.